walletrpc/compact_formats.proto

```diff
@@ -13,9 +13,10 @@ message CompactBlock {
     uint32 protoVersion = 1; // the version of this wire format, for storage
     uint64 height = 2; // the height of this block
     bytes hash = 3;
-    uint32 time = 4;
-    bytes header = 5; // (hash and time) OR (full header)
-    repeated CompactTx vtx = 6; // compact transactions from this block
+    bytes prevHash = 4;
```

Comment:
We're intentionally breaking this?

Reply:
Yes, because a) lightwalletd, like the rest of the stack, is in alpha and explicitly allowed to do this, and b) because it forces everyone to regenerate their lightwalletd servers, ensuring they all have `prevHash` fields.

Reply:
Just wanted it on the record. SGTM.